src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.scss

```diff
@@ -14,3 +14,7 @@
 .indent {
   margin-left: 40px;
 }
+
+.font {
```

Comment:
what happens without this?

Reply:
The font within the mat-tab-group is different. It looks bolded and the family is different (Roboto, "Helvetica Neue", sans-serif;)

<img width="721" alt="Screen Shot 2020-06-18 at 10 59 48 AM" src="https://user-images.githubusercontent.com/16004905/85055939-41a5ac80-b153-11ea-8f0e-ed6a6555dee6.png">